Ullaakut/Bloggo
# Config improvements

## Goal of this PR

Fixes #3 

This PR makes it possible to override the default configuration, but also removes some legacy configuration variables and improves the documentation.

This will make it easier to use Bloggo for people that want to use their own database, to customize the API, etc.

## How to test it

* Run `docker-compose up`
* Ensure that the configuration is the default configuration
* Update the environment variables for configuration in `docker-compose.yml`
* Run `docker-compose stop && docker-compose run`
* Ensure that the configuration is now different and properly updated
